invenio_app_ils/config.py

```diff
@@ -927,18 +927,6 @@ def _(x):
 ILS_INDEXER_TASK_DELAY = timedelta(seconds=2)
 """Trigger delay for celery tasks to index referenced records."""
 
-# Accounts REST
-# ==============
-ACCOUNTS_REST_READ_USER_PROPERTIES_PERMISSION_FACTORY = backoffice_permission
```

Comment:
Why are those removed?

Reply:
Well spotted! I think this is a left over of when @kprzerwa was working on `accounts-rest` module but we abandoned it.
@kprzerwa can you please check?

Reply:
I don't think it was me working on it, I think it was Zach :) but I checked, we don't use invenio-accounts-rest I think it is OK to remove it